app/controllers/finance/lead_providers_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class Finance::LeadProvidersController < Finance::BaseController
+  skip_after_action :verify_authorized
+  skip_after_action :verify_policy_scoped
+
+  def index
+    @lead_providers = LeadProvider.all
+  end
+
+  def show; end
```

Comment:
If we really want to keep the pundit methods (`verify_authorized` and `verify_policy_scoped`), then we should use them here. In particular, scope in the index, and authorize in show. Also, in show, should we load a lead provider by id?

Reply:
Also also, this does not have an existing route, so we will need to either add it, or remove this action.

Reply:
You are right, scope is not required as every user will be able to see all lead providers, at least for now